mon-pix/tests/acceptance/authentication/signup-test.js

```diff
@@ -101,6 +105,10 @@ module('Acceptance | authentication | Signup', function (hooks) {
         assert.dom(signupHeading).exists();
         const loginButton = screen.queryByRole('link', { name: t('pages.sign-up.actions.login') });
         assert.dom(loginButton).doesNotExist();
+        const otherAuthenticationProvidersTitle = screen.queryByText(
+          t('components.authentication.other-authentication-providers.signup.heading'),
+        );
+        assert.dom(otherAuthenticationProvidersTitle).doesNotExist();
```

Comment:
Même remarque que plus haut
```suggestion
        assert.notOk(otherAuthenticationProvidersTitle);
```